packages/react-hig/src/adapters/TextFieldAdapter.js

```diff
@@ -139,7 +148,7 @@ TextFieldComponent.propTypes = {
   value: PropTypes.string
 };
 
-TextFieldComponent.__docgenInfo = {
+TextField.__docgenInfo = {
```

Comment:
same reference here. Shouldn't this be to the TextFieldComponent? otherwise we will have an issue in Storybook with showing props.